apps/src/templates/sectionAssessments/MultipleChoiceDetailsDialog.jsx

```diff
@@ -43,6 +66,28 @@ class MultipleChoiceDetailsDialog extends Component {
           style={styles.instructions}
           dangerouslySetInnerHTML={{ __html: renderedMarkdown }}
         />
+        {(questionAndAnswers.answers && questionAndAnswers.answers.length > 0) &&
+          <div>
+            {questionAndAnswers.answers.map((answer, index) => {
+              return (
+                <div key={index} style={styles.answerBlock}>
+                  <div style={styles.iconSpace}>
+                    {answer.correct &&
+                      <FontAwesome icon="check-circle" style={styles.icon}/>
+                    }
+                    {!answer.correct && (<span>&nbsp;</span>)}
+                  </div>
+                  <div style={styles.answerLetter}>{answer.letter}</div>
+                  <div
+                    style={styles.answers}
+                    dangerouslySetInnerHTML={{ __html: processMarkdown(answer.text, { renderer }) }}
```

Comment:
I'm a little disoriented here. Does this mean we support markdown in student-supplied answers?

Reply:
Nope, the answer options for multiple choice questions (which we create in levelbuilder) can be in markdown. Nothing here is user input and should be fairly safe as far as `dangerouslySetInnerHtml` goes.

Reply:
Oh - markdown in the text associated with multiple-choice answers.